lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1447,16 +1448,48 @@ struct LazyLocationListener : public OpBuilder::Listener {
 };
 } // end anonymous namespace
 
+/// Remove the DontTouch annotation and return true, if it exists.
+/// Ignore DontTouch that will apply only to a subfield.
+static bool removeDontTouch(ArrayAttr &annotations) {
```

Comment:
I don't like the idea of scattering don't touch, then immediately removing them, but maybe it's fine if we move to lowerAnno soon.

Reply:
Yes, i agree. I will try to merge your lowerAnno pass asap.